include/text.h

```diff
@@ -59,7 +65,7 @@ struct TextPrinterSubStruct
     u8 fontId:4;  // 0x14
     bool8 hasPrintBeenSpedUp:1;
     u8 unk:3;
-    u8 downArrowDelay:5;
+    u16 downArrowDelay:13;
     u8 downArrowYPosIdx:2;
     bool8 hasFontIdBeenSet:1;
     u8 autoScrollDelay;
```

Comment:
```suggestion
    u16 downArrowDelay:13;
    u16 downArrowYPosIdx:2;
    16 hasFontIdBeenSet:1;
    u8 autoScrollDelay;
```
need to change the next 2 members as well.